src/components/home.css

```diff
@@ -1,39 +1,85 @@
+/* page container */
+.page-container {
+    background-color: #24323F
```

Comment:
This container should take care of keeping content on the middle. So add `max-width: something you like px` so the content isnt on the edges of the screen. Then add `margin: 0 auto` to center this container. After this you can remove all the horizontal padding from classes like `grid-container` etc